pkg/engine/jmespath/functions.go

```diff
@@ -834,6 +834,14 @@ func jpLabelMatch(arguments []interface{}) (interface{}, error) {
 	return true, nil
 }
 
+func ToBoolean(input string) bool {
+	if input == "true" {
```

Comment:
This is not what we want, we want a real conversion here.

Reply:
Maybe use [this](https://www.golangprograms.com/string-to-boolean-data-type-conversion-in-go.html) ?

Reply:
@eddycharly And any input other than "true" or "false" should result in nil?

Reply:
I have implemented this